rajawali/src/main/java/org/rajawali3d/Object3D.java

```diff
@@ -371,13 +371,18 @@ public boolean isContainer() {
 	}
 
 	/**
-	 * Maps screen coordinates to object coordinates
-	 *
-	 * @param x
-	 * @param y
-	 * @param viewportWidth
-	 * @param viewportHeight
-	 * @param eyeZ
+	 * Positions the object in the plane of the screen/viewport (z = 0), with
+     * x and y specified in GL screen coordinates (origin at bottom left);
```

Comment:
Yep, noticed that too. I'm guessing the "no inconsistent whitespace" guideline in CONTRIBUTING.md means try to maintain whatever formatting style is prevalent in each file?

Also, should I be doing something to squash commits after I push them to my fork? Or should I only ever rewrite history on my local repo?

WIll push the whitespace fix momentarily...